NAV_Algorithms/NAV_tuning_parameters.h

```diff
@@ -25,6 +25,8 @@
 #ifndef NAV_ALGORITHMS_NAV_TUNING_PARAMETERS_H_
 #define NAV_ALGORITHMS_NAV_TUNING_PARAMETERS_H_
 
+#define USE_EARTH_INDUCTION_DATA_COLLECTOR 0
```

Comment:
It seems that this is disabled by default. I prefer having less precompiler flags to keep readability complexity low.

Reply:
kept because required for Simulations.